lib/katello/plugin.rb

```diff
@@ -171,4 +171,11 @@
                    :partial => 'smart_proxies/show/storage',
                    :onlyif => proc { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) || proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) }
   end
+
+  RemoteExecutionFeature.register(:katello_package_install, N_("Katello: Install Package"), :description => N_("Install package via Katello interface"), :provided_inputs => ['package'])
```

Comment:
What makes, for example, package installation specific to Katello? I realize we have UI and other precedence for it, but if I am using say Foreman, Foreman Docker and Remote Execution -- why can't I go use a package install feature on those hosts to install a package I need?